starboard/android/shared/BUILD.gn

```diff
@@ -379,6 +379,11 @@ static_library("starboard_platform") {
   if (sb_evergreen_compatible_use_libunwind) {
     deps += [ "//third_party/llvm-project/libunwind:unwind_starboard" ]
   }
+
+  cflags = [
```

Comment:
This reads really unusual, i don't know that it does. Force-include atomic headers ?